src/components/Nav.jsx

```diff
@@ -0,0 +1,23 @@
+import { Link } from "react-router-dom"
+
+
+function Nav ({topics}) {
+    
+    return (
+        <>  
+            <Link to="/">
+                <button>home</button>
+            </Link>
+            {topics.map((topic) => {
+                return(
+                        <Link to={topic.slug} key={topic.slug}>
+                            <button>{topic.slug}</button>
```

Comment:
This could be confusing for screen readers, if you want it to look like a button I would rely on CSS